parts/windows/windowscsehelper.ps1

```diff
@@ -67,10 +67,14 @@ $global:WINDOWS_CSE_ERROR_GPU_DRIVER_INSTALLATION_URL_NOT_EXE=61
 
 # NOTE: KubernetesVersion does not contain "v"
 $global:MinimalKubernetesVersionWithLatestContainerd = "1.28.0" # Will change it to the correct version when we support new Windows containerd version
-$global:StableContainerdPackage = "v1.6.21-azure.1/binaries/containerd-v1.6.21-azure.1-windows-amd64.tar.gz"
+$global:StableContainerdPackage = "v1.7.21-azure.1/binaries/containerd-v1.7.21-azure.1-windows-amd64.tar.gz"

```

Comment:
out of scope of this PR?

Reply:
It's used for test to log error message. Will change it back after the test.